src/components/MaplibreMap.jsx

```diff
@@ -321,21 +303,21 @@ const MaplibreMap = ({ view, setView }) => {
         mapStyle="https://demotiles.maplibre.org/style.json"
         onLoad={handleMapLoad}
         onMoveEnd={handleMoveEnd}
-        onResize={handleResize}
         onClick={handleClick}
         onMouseEnter={handleMouseEnter}
         onMouseLeave={handleMouseLeave}
         interactiveLayerIds={['sites-cluster-layer', 'sites-unclustered-layer']}
-        // disable the default attribution
         attributionControl={false}
       >
         {mapAndTableControlsWrapper}
-        <AttributionControl
-          compact={true}
-          customAttribution="Source: Esri, Maxar, GeoEye, Earthstar Geographics, CNES/Airbus DS, USDA, USGS, AeroGRID, IGN, and the GIS User Community"
-          position={isDesktopWidth ? 'bottom-right' : 'top-right'}
-        />
-        {isDesktopWidth ? <NavigationControl showCompass={false} position="bottom-right" /> : null}
+        {isDesktopWidth && (
+          <AttributionControl
+            compact={true}
+            customAttribution="Source: Esri, Maxar, GeoEye, Earthstar Geographics, CNES/Airbus DS, USDA, USGS, AeroGRID, IGN, and the GIS User Community"
+            position={'bottom-right'}
+          />
+        )}
```

Comment:
I dont see removing this in mobile view in the requirements. We might have a legal/contractual obligation to always show this somehow. Did the decision to remove this for mobile view get OKed from someone in-the-know about contractual obligations?

Reply:
Hi @mmnoo , this is not stated in the requirement, I ask this in the spk-mermaid channel, this might be just temporary based on @alanjleonard feedback, since it is blocking a new button placement on the right.